rules/integrations/gcp/credential_access_gcp_kubernetes_secrets_enumeration.toml

```diff
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2021/06/06"
+maturity = "production"
+updated_date = "2021/06/06"
+
+[rule]
+author = ["Elastic","Austin Songer"]
+description = """
+Identifies the unauthorized enumeration of kubernetes secrets by a user.
```

Comment:
How we differentiate the enumeration from legitimate activity? If we can't, we need to reformulate this.

Reply:
Also, we need to explain why the this is critical, a little context of what can be done using these secrets would be nice here.